README.md

```diff
@@ -84,11 +103,31 @@ final class AppDelegate: ApplicationDelegate {
 		libwayland-dev \
 		libcairo2-dev \
 		libpango1.0-dev \
-		libglib2.0-dev
+		libglib2.0-dev \
+ 		libclang-15-dev
```

Comment:
If there's a dedicated section talking about installation libclang-dev there's probably no room for it to be here.